__tests__/app.test.js

```diff
@@ -41,13 +41,56 @@ describe('GET /api', () => {
         .expect(200)
         .then((response)=>{
            const res=response.body 
-           
-            for(const key in res){
-                
-            expect(res[key]).toEqual(endPoints[key].description)
-        }
+                       
+            expect(res).toEqual(endPoints)
+        
         })
         
     });
     
-});
\ No newline at end of file
+});
+
+describe('GET /api/articles/:article_id', () => {
+
+    test('should respond with an object containing specific article ', () => {
+        const article_id= 3
+        return request(app).get(`/api/articles/${article_id}`)
+        .expect(200)
+        .then(({body})=>{
+            expect(body.article).toEqual(
+            expect.objectContaining(
+                {
+                  title: "Eight pug gifs that remind me of mitch",
+                  topic: "mitch",
+                  author: "icellusedkars",
+                  body: "some gifs",
+                  article_img_url:
+                    "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                })
+           )
+
```

Comment:
We need to check that the article id that is returned is 3.

Reply:
fixed :)